frontend/composables/useMenuEntriesState.ts

```diff
@@ -3,8 +3,11 @@ import { IconMap } from "~/types/icon-map";
 const createMenuEntry = (label: string, basePath: string, iconURL: string) => {
   const { locale } = useI18n();
   const router = useRouter();
+  // here, afaik, this is returning NaN as the id of a given organization is alphanumeric -- not a number.
+  // added the raw id which can be used with the routeURL , while maintaining the number version
+  const idStr = router.currentRoute.value.params.id;
   const id = Number(router.currentRoute.value.params.id);
```

Comment:
Ya this is a holdover from prior setups where the route URL was an integer. Nice catch! Let's just set id to your value - without the `Number()` :)